app/web/features/profile/view/UserOverview.tsx

```diff
@@ -92,6 +98,17 @@ export default function UserOverview({
       <div className={classes.wrapper}>
         <Typography variant="h1" className={classes.intro}>
           {user.name}
+          {user.hasStrongVerification ? (
```

Comment:
has this function been implemented in the serveside?

Reply:
Yes it has.